### ISSUE TITLE
best way to pass custom config settings to chimp?

### ISSUE THREAD

**Issue by gesweind** [xolvio/chimp#331]
I am a chimp newb, so bear with me.  I would like to be able to put my own config settings (for example a backend service URL that I need to hit before testing) in the chimp config file.  Is there a clean way to do this so that I can access these settings within my tests?

**Comment by samhatoum** [comment]
Yep :)

https://chimp.readme.io/docs/command-line-options

**Comment by gesweind** [comment]
Yes, that's what I want to use.  I read that page, but it's not clear to me how I can access those settings within my step definitions.

**Comment by samhatoum** [comment]
oh my bad, I misread what you said.

1. You can use environment variables. 

You can run chimp with `HELLO=world chimp...` and in your steps you can use `process.env.HELLO`

2. Undocumented "feature" that *should' work (let me know)

pass in a param in the config file / command line and then this construct to access it:

`process.env['chimp.nameOfYourParameter']`

**Comment by gesweind** [comment]
Thank you! Exactly what I was trying to do (and it worked)!

**Comment by gesweind** [comment]
That is, I passed in the param via the config file I was already using - that worked.

**Comment by samhatoum** [comment]
Awesome, I'll keep this open to add the docs for this option. Seems useful

**Comment by AntonioRecaldeRusso** [comment]
Where can we find the documentation for ALL of chimp.js' config/cli options? I have to look at the source code every time. The website is not even superficial about it.

**Comment by samhatoum** [comment]
@AntonioRecaldeRusso you're right this is a pain. I'd love some help in doing this. @deanius and I have started talking about it here: https://github.com/xolvio/chimp/pull/447

Any takers?

**Comment by Windso** [comment]
I'm using the [grunt-chimp npm package](https://www.npmjs.com/package/grunt-chimp). I'd like to set an environment variable from grunt on the command line. @samhatoum any idea how to do this? Something like this:
`grunt myTaskThatRunsChimp --env=myDevServer`
